src/transformers/__init__.py

```diff
@@ -382,6 +382,12 @@
         "TransfoXLCorpus",
         "TransfoXLTokenizer",
     ],
+    "models.trillsson_efficient": [
+        "TRILLSSON_EFFICIENT_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "Trillsson_efficientConfig",
+        "Trillsson_efficientFeatureExtractor",
```

Comment:
Since `Trillsson_efficientFeatureExtractor` requires that `is_speech_available()`, it should not be added to the import list here, but rather [here](https://github.com/huggingface/transformers/blob/main/src/transformers/__init__.py#L693)! 
The same goes with the actual import